README.adoc

```diff
@@ -1,13 +1,25 @@
 # Grails Testing Support
 
-image:https://github.com/grails/grails-testing-support/actions/workflows/gradle.yml/badge.svg["Java CI", link: "https://github.com/grails/grails-testing-support/actions/workflows/gradle.yml"]
+image:https://img.shields.io/badge/Revved%20up%20by-Develocity-06A0CE?logo=Gradle&labelColor=02303A["Revved up by Develocity", link: "https://ge.grails.org/scans"]
+image:https://github.com/grails/grails-testing-support/actions/workflows/gradle.yml/badge.svg?event=push["Java CI", link: "https://github.com/grails/grails-testing-support/actions/workflows/gradle.yml"]
+image:https://github.com/grails/grails-testing-support/actions/workflows/groovy-joint-workflow.yml/badge.svg?event=push["Groovy Joint Validation Build", link: "https://github.com/grails/grails-testing-support/actions/workflows/groovy-joint-workflow.yml"]
 
 Documentation is available at link:https://testing.grails.org[https://testing.grails.org/]
 
-Grails 3.x branch is https://github.com/grails/grails-testing-support/tree/1.1.x[1.1.x].
+Documentation SNAPSHOT is available at link:https://testing.grails.org/snapshot/guide/index.html[https://testing.grails.org/snapshot/guide/index.html]
 
-Grails 4.0.x branch is https://github.com/grails/grails-testing-support/tree/2.1.x[2.1.x].
+Grails 7.0 branch is https://github.com/grails/grails-testing-support/tree/4.0.x[4.0.x]
+
+Grails 6.2 branch is https://github.com/grails/grails-testing-support/tree/3.2.x[3.2.x]
+
+Grails 6.1 branch is https://github.com/grails/grails-testing-support/tree/3.1.x[3.1.x]
+
+Grails 6 branch is https://github.com/grails/grails-testing-support/tree/3.0.x[3.0.x]
 
 Grails 5 branch is https://github.com/grails/grails-testing-support/tree/2.4.x[2.4.x] which includes support for Groovy 3.
 
+Grails 4.0.x branch is https://github.com/grails/grails-testing-support/tree/2.1.x[2.1.x].
+
+Grails 3.x branch is https://github.com/grails/grails-testing-support/tree/1.1.x[1.1.x].
+
 Please checkout https://github.com/grails/grails-testing-support/releases[release notes] for more information.
```

Comment:
Is "checkout" right in this context or should it be "check out". (Intellij is complaining)

Reply:
"Check out" is correct.  I am going to change it to "see", since check out is informal.